src/views/OrderPage/OrderPage.tsx

```diff
@@ -0,0 +1,51 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import * as Style from './index';
+import { Header } from '../../shared/components';
+import * as Components from '../../modules/order';
+import Topping from 'modules/order/models/Topping';
+
+const OrderPage: React.FC = () => {
+  const [price, setPrice] = useState(0.0);
+
+  const toppings = [
```

Comment:
This can be moved outside the functional component so it is not recreated every time the component re-renders. You can move it below the imports and above `const OrderPage`.